### PR TITLE
401 tutorial experimental

### PR DESCRIPTION
**Pull request by GabrieleRoncolato**
Implementation of a 401 tutorial for workflow interface

### CODE REVIEW

**Review by psfoley — COMMENTED** [review]
Thank you @GabrieleRoncolato, @Albio-dev, @Rickybolo99, @leofracca, and @MDV-droid for the contribution! To summarize, this combines the examples from the prior workflow interface notebook examples and demonstrates them running together? It looks like a lot of files got included (datasets, metaflow artifacts, final model, etc.) by mistake. Once those are removed, I think this is ready to merge 🚀 

Also, could you retroactively sign the DCO per [these instructions](https://github.com/securefederatedai/openfl/pull/862/checks?check_run_id=14981150379)? Thanks!

**Comment by GabrieleRoncolato** [comment]
Yes, this example combines the prior workflow interface examples together.
I have read the guide regarding how to sign the DCO, however in our case we need to sign each commit individually, is there a guide regarding how to do that?
Once we have solved this issue we will also proceed with removing the additional files, thank you!

**Comment by GabrieleRoncolato** [comment]
We have updated the pull request by removing the unnecessary files and signing all commits as required, we have also included an authors section in the notebook.
Being unable to sign each commit using a different key, @leofracca verified all commits.

File: openfl-tutorials/experimental/requirements_workflow_interface.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-dill==0.3.6
-metaflow==2.7.15
-ray==2.2.0
+dill==0.3.6
+metaflow==2.7.15
+ray==2.2.0
+numpy==1.21.6
```
Comment (psfoley) [inline openfl-tutorials/experimental/requirements_workflow_interface.txt:4]: Is it necessary to pin the numpy version?

Comment (GabrieleRoncolato) [reply openfl-tutorials/experimental/requirements_workflow_interface.txt:4]: We were having dependencies issues with the latest numpy version while combining the previously existing examples, pinning it allowed our 401 tutorial to run successfully

File: openfl-tutorials/experimental/Workflow_Interface_401_MNIST_Aggregator_Validation_Ray_Watermarking.ipynb [hunk 2]
```diff
@@ -0,0 +1,920 @@
...
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/intel/openfl/blob/develop/openfl-tutorials/experimental/Workflow_Interface_401_MNIST_Aggregator_Validation_Ray_Watermarking.ipynb)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8f28c451",
+   "metadata": {},
+   "source": [
+    "Authors:\n",
```
Comment (psfoley) [inline openfl-tutorials/experimental/Workflow_Interface_401_MNIST_Aggregator_Validation_Ray_Watermarking.ipynb]: The author list should be removed considering this is a consolidation of prior work. All coauthors will be credited when OpenFL 1.6 is released (see our [OpenFL 1.5 release announcement](https://github.com/securefederatedai/openfl/releases/tag/v1.5) for reference)

**Review by psfoley — CHANGES_REQUESTED** [review]
Once the remaining review comment is addressed, this is ready to merge!

File: openfl-tutorials/experimental/Workflow_Interface_401_MNIST_Aggregator_Validation_Ray_Watermarking.ipynb (hunk 2 above)
Comment (leofracca) [reply openfl-tutorials/experimental/Workflow_Interface_401_MNIST_Aggregator_Validation_Ray_Watermarking.ipynb]: We removed the author list with the last commit

**Review by psfoley — APPROVED** [review]

**Pull request merged**